### ISSUE TITLE
More investigation around inductors

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-common#785]
From https://github.com/phetsims/circuit-construction-kit-common/issues/774#issuecomment-973318310, we switched to 1 inductor.  This issue is about working toward the root cause of the multiple inductor problems.  May not need to be done before 1.0.

We would like to try higher resistances in the wires to see if it reduced the problem?  But @kathy-phet says the priority is getting the sign changes fixed.  If some corner cases like this have bad behavior, we may have to list it as a known issue.

@samreid will also check if there is a sudden spike in the inductor values that is causing the problem.

Do we need the initial bias step?

**Comment by samreid** [comment]
On hold until we address the phet-io specific issues.

**Comment by samreid** [comment]
I don't think this should be part of the current sprint, @arouinfar can I remove it from the milestone?

**Comment by kathy-phet** [comment]
I agree this isn't a priority for current Jan 2022 sprint on PHET-iO.

**Comment by samreid** [comment]
We decided this wouldn't impact any of the DC milestones.
